tests/bud/base-with-arg/Containerfile

```diff
@@ -0,0 +1,24 @@
+FROM --platform=$TARGETPLATFORM alpine AS build
+
+ARG TARGETARCH
+ARG TARGETPLATFORM
```

Comment:
Is this in the right place?  It should trigger a warning, since it's not used in this stage.  Are we checking that we get the right warnings about unused arguments?

Reply:
We do get a warning but since this is a default arg and its not needed hence removed both of them.